contracts/libraries/LibYieldGeneration.sol

```diff
@@ -93,14 +93,13 @@ library LibYieldGeneration {
     /// @notice Conversion from eth to shares
     /// @param _collateralAmount The amount of collateral to withdraw
     /// @param _yield The yield generation struct
-    function _ethToShares(
+    function _neededShares(
         uint _collateralAmount,
         LibYieldGenerationStorage.YieldGeneration storage _yield
```

Comment:
Can this not be memory instead of storage? We are not writing to _yield here.

Reply:
No it can't because of the nested mapping, But changed only to the needed inputs on commit 89373d2